README.md

```diff
@@ -10,13 +10,19 @@
 
 # Navigation
 
+- [Codesandbox](#Codesandbox)
 - [Installation](#Installation)
 - [Usage](#Usage)
 - [TypeScript](#TypeScript)
 - [Limitations](#Limitations)
 - [API](https://vue-composition-api-rfc.netlify.com/api.html)
 - [Changelog](https://github.com/vuejs/composition-api/blob/master/CHANGELOG.md)
 
+# Codesandbox
+
+Here is a very minimal codesandbox template (with TS) if you wish to play with it online:
```

Comment:
I think having having 2 options, one with TS and one with JS will allow people not familiar with TS to also play with it